comet_core/api_v0.py

```diff
@@ -40,6 +41,53 @@ def ok():  # pylint: disable=invalid-name
     return jsonify({'status': 'ok'})
 
 
+def action_succeeded(custom_msg):
+    """Generate flask markup to send a custom message for
+    the user based on the action on success
+
+    Args:
+        custom_msg (str): custom message for the action
+    Returns:
+        Markup: the message to return to the user when the action succeeded
+    """
+    message = '{0}'.format(custom_msg)
+    return Markup('<h2>%s</h2><br>'
```

Comment:
if we might send in html at some point, should we sanitize the input here

Reply:
the api is the only one that uses this function in this line:
https://github.com/spotify/comet-core/pull/7/files/223dc439b48229232aed07f826beba191e4fcf82#diff-ef8992e04e31436df508e372773412ecR213

Reply:
👍